src/Octoshift/AzureApi.cs

```diff
@@ -49,14 +49,14 @@ private Uri GetServiceSasUriForBlob(BlobClient blobClient)
                 throw new InvalidOperationException("BlobClient object has not been authorized to generate shared key credentials. Verify --azure-storage-connection-key is valid and has proper permissions.");
             }
 
-            var sasBuilder = new BlobSasBuilder()
+            var sasBuilder = new BlobSasBuilder
```

Comment:
weren't you going to revert this change?

Reply:
I did.....